providers/ibm/iam.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2019 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ibm
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+	bluemix "github.com/IBM-Cloud/bluemix-go"
+	"github.com/IBM-Cloud/bluemix-go/api/iampap/iampapv1"
+	"github.com/IBM-Cloud/bluemix-go/api/iamuum/iamuumv2"
+	"github.com/IBM-Cloud/bluemix-go/api/usermanagement/usermanagementv2"
+	"github.com/IBM-Cloud/bluemix-go/session"
+)
+
+type IAMGenerator struct {
+	IBMService
+}
+
+func (g IAMGenerator) loadUserPolicies(policyID string, user string) terraformutils.Resource {
+	var resources terraformutils.Resource
+	resources = terraformutils.NewSimpleResource(
+		fmt.Sprintf("%s/%s", user, policyID),
+		policyID,
+		"ibm_iam_user_policy",
+		"ibm",
+		[]string{})
+	return resources
+}
+
+func (g IAMGenerator) loadAccessGroups(grpID, grpName string) terraformutils.Resource {
+	var resources terraformutils.Resource
+	resources = terraformutils.NewSimpleResource(
+		fmt.Sprintf("%s/%s", grpID, grpName),
+		grpName,
+		"ibm_iam_access_group",
+		"ibm",
+		[]string{})
+	return resources
+}
+
+func (g IAMGenerator) loadAccessGroupMembers(grpID string) terraformutils.Resource {
+	var resources terraformutils.Resource
+	resources = terraformutils.NewSimpleResource(
+		fmt.Sprintf("%s/%s", grpID, time.Now().UTC().String()),
+		time.Now().UTC().String(),
```

Comment:
It's generate every time other IDs, Maybe better take hash or combine.

Some people run terraformer many time and check git diff